superset-frontend/src/dashboard/components/DashboardBuilder/DashboardBuilder.tsx

```diff
@@ -82,52 +89,66 @@ import { useNativeFilters } from './state';
 type DashboardBuilderProps = {};
 
 const StyledDiv = styled.div`
-  display: grid;
-  grid-template-columns: auto 1fr;
-  grid-template-rows: auto 1fr;
-  flex: 1;
-  /* Special cases */
-
-  /* A row within a column has inset hover menu */
-  .dragdroppable-column .dragdroppable-row .hover-menu--left {
-    left: -12px;
-    background: ${({ theme }) => theme.colors.grayscale.light5};
-    border: 1px solid ${({ theme }) => theme.colors.grayscale.light2};
-  }
-
-  .dashboard-component-tabs {
-    position: relative;
-  }
-
-  /* A column within a column or tabs has inset hover menu */
-  .dragdroppable-column .dragdroppable-column .hover-menu--top,
-  .dashboard-component-tabs .dragdroppable-column .hover-menu--top {
-    top: -12px;
-    background: ${({ theme }) => theme.colors.grayscale.light5};
-    border: 1px solid ${({ theme }) => theme.colors.grayscale.light2};
-  }
-
-  /* move Tabs hover menu to top near actual Tabs */
-  .dashboard-component-tabs > .hover-menu-container > .hover-menu--left {
-    top: 0;
-    transform: unset;
-    background: transparent;
-  }
-
-  /* push Chart actions to upper right */
-  .dragdroppable-column .dashboard-component-chart-holder .hover-menu--top,
-  .dragdroppable .dashboard-component-header .hover-menu--top {
-    right: 8px;
-    top: 8px;
-    background: transparent;
-    border: none;
-    transform: unset;
-    left: unset;
-  }
-  div:hover > .hover-menu-container .hover-menu,
-  .hover-menu-container .hover-menu:hover {
-    opacity: 1;
-  }
+  ${({ theme }) => css`
+    display: grid;
+    grid-template-columns: auto 1fr;
+    grid-template-rows: auto 1fr;
+    flex: 1;
+    /* Special cases */
+
+    /* A row within a column has inset hover menu */
+    .dragdroppable-column .dragdroppable-row .hover-menu--left {
+      left: ${theme.gridUnit * -3}px;
```

Comment:
Is this what we'd want if we ever changed the scale of the grid unit? If it just needs to be 1px forever and always, so be it ;)

Reply:
Good point 🤔 We should do a sweep through the styles and verify if some values should be hardcoded rather than be dependent on theme